### PR TITLE
XW-2864 abtest autoplay

### PR DESCRIPTION
**Pull request by idradm**
https://wikia-inc.atlassian.net/browse/XW-2864

### CODE REVIEW

**Comment by idradm** [comment]
Merge after https://github.com/Wikia/app/pull/12518 is done.

File: extensions/wikia/ArticleVideo/scripts/article-video.js [hunk 1]
```diff
@@ -166,6 +169,27 @@ require(['wikia.window', 'wikia.onScroll', 'wikia.tracker', 'ooyala-player'], fu
 				}
 			});
 
+			player.mb.subscribe( OO.EVENTS.PLAYHEAD_TIME_CHANGED, 'featured-video', function(eventName, time, totalTime) {
+				var secondsPlayed = Math.floor(time),
+					percentage = Math.round(time / totalTime * 100);
+
+				if ( secondsPlayed % 5 === 0 && secondsPlayed !== playTime ) {
```
Comment (latata) [inline extensions/wikia/ArticleVideo/scripts/article-video.js:186]: Are we sure we will get at least one event every second?

Comment (latata) [inline extensions/wikia/ArticleVideo/scripts/article-video.js:186]: In the proposal of drop off tracking I proposed to stop sending tracking events when user uses seek bar (look at examples here https://wikia-inc.atlassian.net/wiki/display/PREMIUM/Videos+drop-offs+tracking)

Comment (idradm) [reply extensions/wikia/ArticleVideo/scripts/article-video.js:186]: Under development the event occurred 3 to 5 times per second, that's why it had to be throttled.

Comment (idradm) [reply extensions/wikia/ArticleVideo/scripts/article-video.js:186]: I'd say the more data we have the better. I don't see any value in stopping the events flow.

**Review by latata — APPROVED** [review]

**Pull request merged**